### ISSUE TITLE
'emend.' authors in Bacteria - continued

### ISSUE THREAD

**Issue by aoern** [CatalogueOfLife/data#664]
#650 reported problems in Bacteria related to emend. phrases.
Now the emend. clause has been removed from the authorship string, which is ok.
However, the comma between the author list and the year has also been removed resulting in scientific names like these:
Acinetobacter lwoffii  (Audureau 1940) Brisou & Prévot 1954
Actinocatenispora  Thawai et al. 2006
Actinoplanes  Couch 1950
Alcaligenes faecalis  Castellani & Chalmers 1919
etc.

**Comment by mdoering** [comment]
... which is the correct way to cite authors in the bacterial code.
https://www.ncbi.nlm.nih.gov/books/NBK8808/#A332
https://www.dsmz.de/collection/catalogue/details/culture/DSM-2403
https://en.wikipedia.org/wiki/Acinetobacter_lwoffii

**Comment by yroskov** [comment]
My approach is always to keep authorstrings in the form as they given by data provider.

**Comment by aoern** [comment]
At the moment, fixed policy is applied:

1. The data provider (ITIS) applies the botanical code (comma between author and year).
This has been applied in CoL this far.
"Actinocatenispora	Thawai et al., 2006"

2. The taxa that have have an emend. clause in there authorship string, apply the bacterial code (no comma).
"Actinocatenispora	Thawai et al. 2006"

3. The namePhrase property of these 'emend. cases' 2. applies the botanical code.
"emend. Stackebrandt & Kroppenstedt, 1987"

We definately need a common policy to apply!

**Comment by mdoering** [comment]
indeed. The namePhrase is unstructured, thats why. The author team is a proper list of authors and a year. I was wondering to make emend also a proper author team, but that would have added lots of complexity and code which I didn't see much benefits for. You just pointed out a new benefit I did have in mind

**Comment by DaveNicolson** [comment]
The commas in those strings in ITIS are not specifically tied to a particular nomenclatural Code, but Tom Orrell was the one handling that update (I suspect it's an artifact of how the processing went). They are all intended to follow the International Code of Nomenclature of Prokaryotes (ICNP). Although I do realize that names given in the [current ICNP](https://www.microbiologyresearch.org/content/journal/ijsem/10.1099/ijsem.0.005585) don't have commas before the year, I don't see the word comma in it, so I am not sure whether that's a formal requirement. I've not done a deep dive on this, so happy to be corrected!

**Comment by TonyRees** [comment]
Hi all,

Under the ICZN Code and the ICN Code (zoology and botany), "emend." is not
part of the formal authorship so, at least in IRMNG, I strip it off when
supplied (effectively it is some sort of remark). However under the
prokaryote code it does seem to have some more formal status - at least it
is regularly used in their lists of valid names - so I have presumed it
forms part of the authorship string for those names, and have left it in.
(Checking the relevant Code supports this view, i.e.
Parker CT, Tindall BJ, Garrity GM. 2019. International Code of Nomenclature
of Prokaryotes. Prokaryotic Code (2008 Revision). Int J Syst Evol
Microbiol. 69(1A): S1-S111.
<https://www.ccug.se/documents/taxonomy/prokaryotic_code/2019_Parker-Tindall-Garrity_Prokaryotic%20Code%20-%202008%20Revision.pdf>

Relevant item is here:
------------
 *Citation of the Name of a Taxon whose Circumscription Has Been Emended*
Rule 35 If an alteration of the diagnostic characters or of the
circumscription of a taxon modifies the nature of the taxon, the author
responsible may be indicated by the addition to the author citation of the
abbreviation ‘‘emend.’’ (emendavit) followed by the name of the author
responsible for the change. Example: *Rhodopseudomonas *Czurda and Maresch
1937 emend. van Niel 1944 (see Opinion 49; Judicial Commission [20]).

Reference cited:  20. Judicial Commission. Opinion 49. Conservation of the
generic name *Rhodopseudomonas *Czurda and Maresch emend. van Niel. Int J
Syst Bacteriol 1974;24:551.

[Another example in references:  30. Bøvre K. Proposal to divide the
genus *Moraxella
*Lwoff 1939 emend. Henriksen and Bøvre 1968 into two subgenera –
subgenus *Moraxella
*(Lwoff 1939) Bøvre 1979 and subgenus *Branhamella *(Catlin 1970) Bøvre
1979. Int J Syst Bacteriol 1979;29:403–406.]
------------
Also from the Code,  List 2. Conserved names of genera of prokaryotes
(nomina generum conservanda):
  Conserved generic names (nomina generum conservanda):  Methanococcus
(Approved Lists 1980) emend. Mah and Kuhn 1984
  (etc.)
-----------

So I would favour maintaining instances of "emend." in authorship strings
for prokaryotes, but not elsewhere.

Regards - Tony

**Comment by DaveNicolson** [comment]
https://www.microbiologyresearch.org/content/journal/ijsem/10.1099/ijsem.0.005585
Oren et al. 2023. International Code of Nomenclature of Prokaryotes. Prokaryotic Code (2022 Revision). International Journal of Systematic and Evolutionary Microbiology, Volume 73, Issue 5a. https://doi.org/10.1099/ijsem.0.005585

**Comment by TonyRees** [comment]
Thanks Dave... text associated with Rule 35, and examples as given above, are carried through unchanged to this new release of the Prokaryotic Code. 

Regards - Tony
